microprofile/cors/src/main/java/io/helidon/microprofile/cors/CorsSupportMp.java

```diff
@@ -112,7 +112,7 @@ protected Builder secondaryLookupSupplier(
 
         @Override
         public String path() {
-            return requestContext.getUriInfo().getPath();
+            return "/" + requestContext.getUriInfo().getPath();
```

Comment:
Could not find in docs whether there is a possibility we would get the leading `/` in response. In such a case I would probably try to guard against it, e.g. 
```java
String path = requestContext.getUriInfo().getPath();
if (path.startsWith("/")) {
            return path;
        } else {
            return '/' + path;
        }
```